src/modules/storage/LedgerStorage.ts

```diff
@@ -3501,11 +3501,14 @@ export class LedgerStorage extends Storages {
      * and if an error occurs the `.catch` is called with an error.
      */
     public getBOAStats(): Promise<any[]> {
-        const sql = `SELECT max(height) as height,
-             (SELECT count(*) from transactions) as transactions,
-             (SELECT count(*) from validators) as validators
-            FROM
-                blocks;`;
+        const sql = `
+                SELECT max(height) as height,
+	                (SELECT SUM(total_reward) as total_reward from accounts) as total_reward,
+                    (SELECT SUM(total_frozen) as total_frozen from accounts) as total_frozen,
+                    (SELECT count(*) from transactions) as transactions,
+                    (SELECT count(DISTINCT address) from validators) as validators
+                    FROM
+                    blocks;`;
```

Comment:
It seems like a query that lot of costs.
Can you reduce the DB load?
Is aggregate data not available?